dockers/docker-ptf/Dockerfile.j2

```diff
@@ -139,6 +139,17 @@ RUN ln -s /usr/bin/tcpdump /usr/sbin/tcpdump
 
 RUN mkdir -p /var/log/supervisor
 
+# add dependencies to build gnmi_get
+RUN wget -O go.tgz https://dl.google.com/go/go1.12.linux-amd64.tar.gz \
+    && tar -C /usr/local -xzf go.tgz \
+    && export PATH=$PATH:/usr/local/go/bin \
+    && go version \
+    && mkdir $HOME/go \
```

Comment:
Why in `$HOME`?

Reply:
Actually this can be any working directory for go,go get step will copy gnmi_get at that location. Any other location you suggest?

Reply:
changed to /etc/go